backport/index.ts

```diff
@@ -1,19 +1,7 @@
-import axios, { AxiosRequestConfig } from 'axios';
 import * as core from '@actions/core';
 import { exec } from '@actions/exec';
 import { context } from '@actions/github';
-import { backportRun, ConfigFileOptions } from 'backport';
-
-export const getConfig = async (repoOwner: string, repoName: string, branch: string, accessToken: string) => {
-  const url = `https://raw.githubusercontent.com/${repoOwner}/${repoName}/${branch}/.backportrc.json`;
-  const config: AxiosRequestConfig = {
-    method: 'get',
-    url: url,
-    headers: { Authorization: `token ${accessToken}` },
-  };
-  const resp = await axios(config);
-  return resp.data as ConfigFileOptions;
-};
```

Comment:
It's not needed to get the config manually. This is already handled by the backport tool.